kitsune/gallery/utils.py

```diff
@@ -62,4 +64,5 @@ def check_media_permissions(media, user, perm_type):
 
 
 def get_draft_title(user):
-    return u'Draft for user %s' % user.username
+    return u'Draft for user %s. Created at: %s' % (user.username,
+                                                   datetime.now())
```

Comment:
The error occurs because we used to just call the draft image "Draft for user <username>" and if the image never got uploaded and the draft never got deleted, this image would persist. Causing integrity errors when the user tries to upload a new image because it can't create the intermediate draft step.